riff-raff/app/views/restrictions/form.scala.html

```diff
@@ -0,0 +1,81 @@
+@import b3.vertical.fieldConstructor
+@import helper.CSRF
+@import _root_.restrictions.RestrictionForm
+@(restrictionForm: Form[RestrictionForm], saveDisabled: Boolean)(implicit request: Security.AuthenticatedRequest[AnyContent, com.gu.googleauth.UserIdentity], messages: Messages)
+
+@main("Create Restriction", request) {
+
+    <h2>Create Restriction</h2>
+    <hr/>
+
+    <div class="row">
+        <div class="col-md-7">
+            @b3.form(action=routes.Restrictions.save) {
+            @CSRF.formField
```

Comment:
😄  - `b3` actually has a `formCSRF` that does this for you so I've just switched over.